addons/sms/tests/test_sms_composer.py

```diff
@@ -0,0 +1,118 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.sms.tests.common import SMSCommon, SMSCase
+from odoo.tests import tagged
+from odoo.tools import html2plaintext, plaintext2html
+
+
+@tagged('at_install')
+class TestSMSComposerComment(SMSCommon, SMSCase):
+    """ Test behaviors that are overridden when other modules
+    are installed (e.g., mass_mailing). In these cases,
+    test_mail_sms or test_mail_full should be used."""
+
+    def test_message_post_sms_vs_notification(self):
```

Comment:
Don't really understand all that ... code seems to consider input is text only.

Reply:
The other two tests fail without the fix. This one is to make sure that we do not break other things while fixing them.